modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/TBD
+[id="cnv-vm-wizard-fields-web_{context}"]
+== Virtual machine wizard fields
+
+|===
+|Name |Parameter |Description
+
+|Name
+|
+|Name of the virtual machine. The name can contain lower-case letters (`a-z`), numbers (`0-9`), periods (`.`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name may not contain spaces or special characters.
+
+|Description
+|
+|Optional description field.
+
+|Template
+|
+|Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
+
+.5+|Provision Source
+|PXE
+|Provision virtual machine from PXE menu. Requires a xref:pxebooting[PXE-capable NIC] in the cluster.
+
+|URL
+|Provision virtual machine from an image available from an *HTTP* or *S3* endpoint.
+
+|Container
+|Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`
+
+|Import
+|Import virtual machine from a supported provider.
+
+|Provider
+|VMware
+|List of supported providers
+
+.2+|vCenter instance
+|*Connect to New Instance*
+|Fill in the *vCenter Hostname*, *vCenter User Name*, and *vCenter Password*.
+
+|Existing vCenter instance
+|The wizard tests the connection with the saved vCenter credentials.
+
+|VM to Import
+|
+|A list of virtual machines available in the provider.
+
+|Operating System
+|
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
```

Comment:
s/available in the cluster. This is the primary operating system for the virtual machine. /that is available to use as the operating system for the VM.

s/VMware virtual machine being imported./VMware virtual machine that you import.